content/docs/patterns/generic-method-pattern/index.md

```diff
@@ -126,7 +129,8 @@ codeunit 53100 "WLD BlockCustomer Meth"
     var
         IsHandled: Boolean;
     begin
-        if not ConfirmBlockCustomer(HideDialog) then exit;
+        if not ConfirmBlockCustomer(HideDialog) then 
+            exit;
```

Comment:
This is a bit of a stylistic choice, since they're functionally the same, but the historical guidelines do call for it to be a new line, oddly enough.   The one benefit that trumps easy compact readability for me is that it makes the "exit" action jump out, which can be a good thing.